Scripts/assignment/emme_assignment.py

```diff
@@ -448,7 +438,7 @@ def _specify(self):
                 van.spec,
             ],
             "background_traffic": {
-                "link_component": param.background_traffic,
+                "link_component": "ul3",
```

Comment:
Why?

Reply:
For some reason when accessing link parameters from Network API (as in module _calc_background_traffic) the name of this parameter is "data3" and in assignment the same parameter is called "ul3". I could not find any smart way around this issue when using imported parameters, so decided to hard code "ul3/data3" to Emme-assignment. Are there some drawbacks that I didn't think about?

Reply:
Maybe it could be called ul3 in parameters, but we would replace "ul" with "data" where necessary? I think it would be nice if this would not be hardcoded.